src/js/utils/RuleExporter.js

```diff
@@ -169,12 +171,11 @@ class RuleExporter {
         definition: rulePropertyDefinition,
         selector: rulePropertyJSON.selector,
         attribute:
-          rulePropertyJSON.attribute != null
-            ? rulePropertyJSON.attribute
-            : rulePropertyJSON.type === RulePropertyTypes.STRING
-              ? 'textContent'
-              : 'innerContent',
-        format: rulePropertyJSON.dateTimeFormat,
+          rulePropertyJSON.type == RulePropertyTypes.DATETIME &&
```

Comment:
We cannot remove the logic for dealing with empty attributes in other fields. The logic here should be:

- If attribute came not null, use the attribute name.
- If the attribute came null:
   - If the type is string, use 'textContent'
   - If the type is date, use 'dateTextContent'
   - Otherwise use 'innerContent'